stl/inc/flat_set

```diff
@@ -586,7 +589,7 @@ private:
         }
 
         if (_Where != _End && !_Compare(_Val, *_Where)) {
-            // convert const_iterator to iterator
+            // *_Where is equivalent to _Val; convert _Where to iterator type.
             return _Mycont.begin() + (_Where - _Begin);

```

Comment:
Btw, this conversion can be avoided after using `container::const_iterator` for `iterator` type.